### PR TITLE
Adding neighboring location to victim's reported location

### PR DESCRIPTION
**Pull request by Meghanaskanagal**
Neighboring location to victims reported location is added to the victim's details only when the status is REPORTED and ASSIGNED

### CODE REVIEW

**Comment by invincibleJai** [comment]
Pls update before /after the chnage screenshot/gif as applicable

File: src/Display.tsx [hunk 1]
```diff
@@ -144,7 +153,20 @@ const VictimDetail: React.FC<VictimDetailProps> = props => {
...
+          // Setting the Victim's locations
+          neighbouringLocations.push(jsonData.features[i].place_name);
+        }*/
+
+        // To show a random nearby location to Victim's location.
+        neighbouringLocations.push(jsonData.features[1].place_name);
+        // Setting the Victim's location
         setAddress(jsonData.features[0].place_name);
+
+        setNeighbourAddress(neighbouringLocations.toString());
```
Comment (rottencandy) [inline src/Display.tsx]: Can we omit the use of intermediate `neighbouringLocations` array?
```javascript
setNeighbourAddress(jsonData.features[1].place_name);
```

File: src/Display.tsx [hunk 2]
```diff
@@ -171,24 +193,38 @@ const VictimDetail: React.FC<VictimDetailProps> = props => {
...
                 </FlexItem>
                 <FlexItem>{props.data.numberOfPeople}</FlexItem>
                 <FlexItem>{props.data.victimPhoneNumber}</FlexItem>
-                <FlexItem>{String(props.data.medicalNeeded)}</FlexItem>
+                {props.data.medicalNeeded ? (
+                  <FlexItem>Required.</FlexItem>
+                ) : null}
+                {!props.data.medicalNeeded ? (
+                  <FlexItem>Not Required.</FlexItem>
+                ) : null}
```
Comment (rottencandy) [inline src/Display.tsx]: The additional check is not needed:
```javascript
{props.data.medicalNeeded ?
<FlexItem>Required.</FlexItem>
) :
<FlexItem>Not Required.</FlexItem>
)}
```

**Comment by invincibleJai** [comment]
why we have PR https://github.com/Emergency-Response-Demo/findmyrelative-frontend/pull/14 with the same title ? are they duplicate PRs? pls close one if so else provide proper title/details/trello link

**Pull request merged**